src/main/java/RaiseTechTask/TaskStudentsManagement_final/version/controller/StudentController.java

```diff
@@ -48,41 +47,37 @@ public String getCoursesList(Model model) {
         return "courseList";
     }
 
-    //新規生徒情報登録画面
+    //新規生徒情報登録
     @GetMapping("/newStudent")
     public String newStudent(Model model) {
-        model.addAttribute("studentDetail", new StudentDetail());
+        StudentDetail studentDetail = new StudentDetail();
+        studentDetail.setStudentsCourses(Arrays.asList(new Course()));
+        model.addAttribute("studentDetail", studentDetail);
         return "registerStudent";
     }
-
     @PostMapping("/registerStudent")
     public String registerStudent(@ModelAttribute StudentDetail studentDetail, BindingResult result) {
         if (result.hasErrors()) {
             return "registerStudent";
         }
-        service.postStudent(studentDetail.getStudent());
+        service.postStudent(studentDetail);
         return "redirect:/studentList";
     }
 
-    //新規コース情報登録
-    @GetMapping("/newCourse")
-    public String newCourse(Model model) {
-        List<Student> students = service.searchStudentList();
-        model.addAttribute("students", students);
-        model.addAttribute("course", new Course());
-        return "registerCourse";
+    //生徒情報更新
+    @GetMapping("/upDateStudent/{studentId}")
```

Comment:
updateは２単語でなく１単語ですね。
```suggestion
    @GetMapping("/updateStudent/{studentId}")
```

Reply:
[こちら](06431a7e8a52ec1e9dcf8f935a9ffba5cbf8d560)で修正を行いました